docroot/profiles/custom/cgov_site/modules/custom/cgov_vocab_manager/src/Manager/CgovVocabManager.php

```diff
@@ -22,7 +22,7 @@ class CgovVocabManager {
   /**
    * Generates a rendered breadcrumb that print taxonomy hiererchy levels.
    */
-  public function getTaxonomyBreadcrumb(int $parent_tid, TermStorageInterface $storageController, RendererInterface $renderer) {
+  public function getTaxonomyBreadcrumb(?int $parent_tid, TermStorageInterface $storageController, RendererInterface $renderer) {
```

Comment:
why is this not complaining that `$parent_tid` should come at the end???

Reply:
It's still required, it's just nullable.